.buckconfig

```diff
@@ -44,4 +44,14 @@
     app_platform = android-27
     cpu_abis = armv7, x86
 
+[download]
+    in_build = true
+    max_number_of_retries = 3
+
+[maven_repositories]
```

Comment:
This should be inferred from okbuck via the repositories dsl and put in the okbuck generated buckconfig